src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -7,6 +7,8 @@ import { metricError } from "config/metric-names";
 import { sendAnalytics } from "utils/analytics-client";
 import { AnalyticsEventTypes, AnalyticsTrackEventsEnum } from "interfaces/common";
 
+const TIMEOUT_PERIOD = 30 * 1000;
```

Comment:
nit: could've been `TIMEOUT_PERIOD_MSEC` for clarity

Reply:
renamed to `TIMEOUT_PERIOD_MS`